src/cljs/proton/layers/base.cljs

```diff
@@ -10,3 +12,43 @@
 (defmulti get-keybindings dispatch)
 (defmulti get-keymaps dispatch)
 (defmulti describe-mode dispatch)
+
+(defn register-package-dependencies [key deps]
```

Comment:
That was initially a leftover from a first version. I tried to tie the linter packages to the linter package itself. 

I think we could remove it